server/consumer.go

```diff
@@ -4177,6 +4454,7 @@ func (o *consumer) loopAndGatherMsgs(qch chan struct{}) {
 			pmsg.returnToPool()
 			goto waitForMsgs
 		}
+		// If we are a fresh pin, add a header
```

Comment:
Stray comment or is there missing code here?

Reply:
Stray comment. Code move elsewhere.